doc/source/transformation_over_time.rst

```diff
@@ -0,0 +1,33 @@
+========================
+Managing Transformations over Time
+========================
+
+In applications, where the transformations between coordinate frames are 
+changing over time, prior to using 
+:class:`~pytransform3d.transform_manager.TransformManager` 
+you should consider interpolating between samples.
+
+We can visualize the problem in the figure below. The tranformation graph with 
+3 coordinate systems: W (world), A, B is visualized over time. Each circle 
+represents a sample (measurement) holding the transformation from the parent 
+to the child frame.
+
+Let's assume we want inspect the situation at the timestep tq (q=query). further
+we want to transform points in B to points in A. As shown in the previous example
```

Comment:
The example script does it the other way around. I'd suggest to make this coherent.